src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ByteArrayModelBinder.cs

```diff
@@ -32,23 +64,26 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
             var value = valueProviderResult.FirstValue;
             if (string.IsNullOrEmpty(value))
             {
+                _logger.FoundNoValueInRequest(bindingContext);
+                _logger.DoneAttemptingToBindModel(bindingContext);
                 return Task.CompletedTask;
             }
 
             try
             {
                 var model = Convert.FromBase64String(value);
                 bindingContext.Result = ModelBindingResult.Success(model);
-                return Task.CompletedTask;
             }
             catch (Exception exception)
             {
                 bindingContext.ModelState.TryAddModelError(
                     bindingContext.ModelName,
                     exception,
                     bindingContext.ModelMetadata);
-                return Task.CompletedTask;
             }
+
+            _logger.DoneAttemptingToBindModel(bindingContext);
+            return Task.CompletedTask;
```

Comment:
> The return pattern is changing in a fair number of the model binders

It's changing here and in the Double,Decimal and Float model binders. The change is simple.

Reply:
The change is simple but it caused extra review time because reviewing it (in GitHub) was less simple. And, I would like @rynowak's thoughts on other implications.